GeosUtil/unitconv_mod.F90

```diff
@@ -412,9 +459,32 @@ SUBROUTINE Convert_Spc_Units( Input_Opt, State_Chm, State_Grid, State_Met, &
 
     END SELECT
 
+    !========================================================================
+    ! Additional checks
+    !========================================================================
+
+    ! Test if all species have mixing ratio units (needed for AIRQNT)
+    State_Chm%allSpeciesInDryMixingRatio = (                                   &
+         ALL( State_Chm%Species(:)%Units == KG_SPECIES_PER_KG_DRY_AIR         )&
+         .or.                                                                  &
+         ALL( State_Chm%Species(:)%Units == MOLES_SPECIES_PER_MOLES_DRY_AIR   )&
+    )
```

Comment:
Would it be faster to have this in a separate subroutine and only call it in AIRQNT when the check is needed? Thinking more about this, it could be a function that returns true or false.

Reply:
Good point @lizziel.  I'll see how best to implement it.

Reply:
@lizziel: `State_Chm%allSpeciesInDryMixingRatio` has now been converted to a PRIVATE function in `GeosCore/calc_met_mod.F90` in commit 79d211841.